ui/src/views/Dashboard.vue

```diff
@@ -240,6 +251,18 @@ export default {
   mounted() {
     this.getStats();
   },
+  updated() {
+    var $ = window.jQuery;
+    $('[data-toggle="tooltip"]').tooltip();
+    if (!this.usersPerWanPieChart) {
+      this.usersPerWanPieChart = generatePieChart("#usersPerWanPieChart-pie-chart", {
+         columns: []
+       },{ width: 400, height: 200});
+    }
+    this.usersPerWanPieChart.load({
+      json: this.status.openvpn.roadwarrior.usersPerWanIP
+    });
+  },
```

Comment:
Is this necessary? Other charts don't need this code to handle the tooltip.
`updated()` is a lifecycle hook that needs to be handled with care, since it is invoked *everytime* vuejs update the view

Reply:
we use it elsewhere like this, nethserver-mail-filter is one example. However when I useit in a function for example getStats() the piechart is not loaded :-?

Reply:
[You are right](https://github.com/NethServer/nethserver-mail/blob/c83db346fa6d1c310fb2430c85f3f2c2389fa01d/ui/src/views/Dashboard.vue#L474-L476). No problem, LGTM